docs : Improve README (#81)
vuetifyjs/awesome-vuetify: README.md

# Awesome Vuetify | [![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re)

[<img src="logo.png" align="right" width="101">](https://vuetifyjs.com)

> A curated list of resources for Vuetify.

> [Vuetify](https://vuetifyjs.com) is a no design skills required UI Library with beautifully handcrafted [Vue](https://vuejs.org/) Components.

Reviewer: ```suggestion
[Vuetify](https://vuetifyjs.com/) is a no design skills required Open Source UI Library with beautifully handcrafted Vue Components.
```

If you have something that you think belongs here, feel free to reach out to us in the [community](#community).

## 📢 Community 

> [Discord](https://discord.com/invite/s93b7Fv)

Reviewer: ```suggestion
[Discord](https://community.vuetifyjs.com/)
```

[Twitter](https://twitter.com/vuetifyjs)
[Facebook](https://www.facebook.com/vuetifyjs)

> [StackOverflow](https://stackoverflow.com/questions/tagged/vuetify.js)

Reviewer: 📄 [Docs](https://vuetifyjs.com/)
🚨 [Issues](https://issues.vuetifyjs.com/)
🏬 [Store](https://store.vuetifyjs.com/)
🎮 [Play](https://play.vuetifyjs.com/)
💬 [Discord](https://community.vuetifyjs.com/)
🐦 [Twitter](https://twitter.com/vuetifyjs)
👽 [Reddit](https://reddit.com/r/vuetifyjs)

## 📄Contents
  - [Tutorials](#tutorials)
  - [Projects using Vuetify](#projects)
    - [🏢 Open source solutions](#open-source)
    - [💡Example projects](#examples)
    - [🖼️ Themes](#themes)
  - [Contribute](#contribute)

> ### Tutorials
> | Framework                     | Link                                                                                                          |
> |-------------------------------|---------------------------------------------------------------------------------------------------------------|
> | Firebase 🔥                   | [Youtube](https://www.youtube.com/watch?v=dIkPb8krORU)                                                        |
> | Storybook 📖                  | [Documentation](https://storybook.js.org/recipes/vuetify)                                                     |
> | NUXT 💎                       | [Youtube](https://www.youtube.com/watch?v=vVVgB5fZJy0)                                                        |
> | Laravel                       | [Blog](https://codersdiaries.com/blog/laravel-vuetify)                                                        |
> | Desktop app (Vuetify + NW.js) | [Youtube](https://www.youtube.com/playlist?list=PLmJs3lfUmCdT9MyG60Oo6HM7xAn79vwZ0)                           |
> | Service workers               | [Youtube](https://medium.com/@dougallrich/give-users-control-over-app-updates-in-vue-cli-3-pwas-20453aedc1f2) |
> | Custom library using Vuetify  | [Youtube](https://github.com/menteora/vuetify-extra)                                                          |

Reviewer: ```suggestion
```

Since this has been out of date for so long I'd like to put a keen focus on only Vuetify related projects, particular Vuetify 3 stuff.
